sorc/global_cycle.fd/read_write_data.f90

```diff
@@ -1512,6 +1577,22 @@ SUBROUTINE READ_DATA(LSOIL,LENSFC,DO_NSST,INC_FILE,IS_NOAHMP, &
  STCFCS = RESHAPE(DUMMY3D, (/LENSFC,LSOIL/))
  ENDIF
 
+ IF (PRESENT(SLCINC)) THEN
```

Comment:
I'm 99% sure that the files created by JEDI use the variable name for the increment (not "*_inc").You can delete the SLCINC and STCINC options, and just use the STC and SLC options above (there's no special snow increment option, which makes me think I'm right). 

If you have done this, and developed the tests using files you created yourself (for synthetic experiments), then I think we need to generate some actual JEDI files and use those instead. I can help with this.

Reply:
Discussed this issue with Clara via email on 02/23/2024. For now, the soil_xainc.00$N files are created using the output from interpolated GSI increments. Will contact Mike to create some real-JEDI derived increments.